mqtt-format/src/v5/variable_header.rs

```diff
@@ -101,10 +103,24 @@ macro_rules! define_properties {
                     Property::$name(value)
                 }
             }
+
+            #[cfg(test)]
+            #[tokio::test]
+            async fn $testfnname () {
```

Comment:
Please collect them in their own mod

Reply:
This does not work because I cannot re-define the module several times.